docs/en/connector-v2/source/Hive.md

```diff
@@ -33,11 +33,12 @@ Read all the data in a split in a pollNext call. What splits are read will be sa
 
 ## Options
 
-| name           | type   | required | default value |
-|----------------|--------|----------|---------------|
-| table_name     | string | yes      | -             |
-| metastore_uri  | string | yes      | -             |
-| common-options |        | no       | -             |
+| name            | type   | required | default value |
+|-----------------|--------|----------|---------------|
+| table_name      | string | yes      | -             |
+| metastore_uri   | string | yes      | -             |
+| read_partitions | string | no       | -             |
```

Comment:
array is better? For example `read_partitions=[par1=true, par1=false/par2=true]`